File: Front_Angular_Pathfinder/src/app/core/models/CronogramaDTO.ts
import { ItemCuadernilloDTO } from "./ItemCuadernilloDTO";

export interface CronogramaDTO {
    CronId: number;
    CronEstaHecho: boolean;
    CronFechaInicio: Date;
    CronFechaFin: Date;
    CronItem: ItemCuadernilloDTO;
}

File: Back_Net8_Pathfinder2/Core/DTO/CronogramaDTO.cs
namespace Core.DTO;

public class CronogramaDTO
{
    public int CronId { get; set; }
    public bool CronEstaHecho { get; set; }
    public DateTime CronFechaInicio { get; set; }
    public DateTime CronFechaFin { get; set; }
    public ItemCuadernilloDTO CronItem { get; set; }
}